warehouse/query-core/src/main/java/datawave/query/transformer/UniqueTransform.java

```diff
@@ -132,23 +103,6 @@ public void updateConfig(UniqueFields uniqueFields, QueryModel model) {
                 log.trace("unique fields: " + this.uniqueFields.getFields());
             }
         }
-        setModelMappings(model);
-    }
-
-    /**
-     * Set the query model from which the reverse query mappings are pulled.
-     *
-     * @param model
-     *            The query model
-     */
-    private void setModelMappings(QueryModel model) {
-        if (model != null) {
```

Comment:
This whole mapping is already done in the DefaultQueryPlanner when setting up the query fields.  This was redundant.